### PR TITLE
1) added my Let's Split keymap 2) improved Alpha keymap

### PR DESCRIPTION
**Pull request by PyrooL**
Added /keyboards/lets_split/keymaps/pyrol because I figured it had deviated enough from default to warrant a pull, plus I want to share with the community.

Made changes to alpha/keymaps/default/keymap.c after some testing.

### CODE REVIEW

**Comment by PyrooL** [comment]
I see there are a lot of errors involving other Let's Split keymaps, but mine also has plenty. Please let me know what is a warning being treated as an error and what needs to be fixed. I have flashed this keymap successfully on both my Let's Splits without problems.

File: keyboards/alpha/keymaps/default/keymap.c [hunk 1]
```diff
@@ -1,76 +1,46 @@
...
-void matrix_init_user(void) {
-}
-
-void matrix_scan_user(void) {
-}
-
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
-void led_set_user(uint8_t usb_led) {
+	LAYOUT(
```
Comment (drashna) [inline keyboards/alpha/keymaps/default/keymap.c]: Just a friendly suggestion, you may want to number the layers, for easy reference.

File: keyboards/lets_split/1 [hunk 2]
```diff
@@ -0,0 +1,23 @@
+cat: .: Is a directory
```
Comment (drashna) [inline keyboards/lets_split/1]: Could you remove this file?

File: keyboards/lets_split/config.h [hunk 3]
```diff
@@ -21,4 +21,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #include "config_common.h"
 
+#undef RGBLED_NUM
```
Comment (drashna) [inline keyboards/lets_split/config.h]: This should be in your keymap's config.h, NOT the keyboards.

If you could remove this from this file, and add it to your keymap's config.h (eg, `/keyboards/lets_split/keymaps/pyrol/config.h`)

File: keyboards/lets_split/keymaps/DE_simple/keymap.c [hunk 4]
```diff
@@ -1,155 +0,0 @@
-#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/lets_split/keymaps/DE_simple/keymap.c]: Was there a reason to remove this?

File: keyboards/lets_split/keymaps/default/keymap.c [hunk 5]
```diff
@@ -1,4 +1,6 @@
-#include QMK_KEYBOARD_H
+#include "lets_split.h"
```
Comment (drashna) [inline keyboards/lets_split/keymaps/default/keymap.c]: Could you revert the changes to this keymap? 

And ideally, you should create your own folder for the keymap.  Eg: https://docs.qmk.fm/#/newbs_building_firmware

Comment (PyrooL) [reply keyboards/lets_split/keymaps/default/keymap.c]: ah whoops. I was editing default until i moved everything to keymaps/pyrol

File: keyboards/alpha/keymaps/default/keymap.c (hunk 1 above)
Comment (drashna) [reply keyboards/alpha/keymaps/default/keymap.c]: https://github.com/qmk/qmk_firmware/blob/master/keyboards/lets_split/keymaps/default/keymap.c#L9-L14

Though, you can use an enum instead (kind of like with the keycodes)

And for the layer:
https://github.com/qmk/qmk_firmware/blob/master/keyboards/lets_split/keymaps/default/keymap.c#L41

**Comment by PyrooL** [comment]
Ended up removing my fork because my upstream sync was awful. Reopening a pull request with the new repo now.
